doc/mock_example/input.yaml

```diff
@@ -69,15 +84,18 @@ sources:
     }
     MWS_MAIN: {
        target_name: MWS_MAIN,
+       truth_name: MWS_MAIN,
        mag_bright: 15.0, 
        mag_faint_filler: 19.0, 
        mag_faintest: 20.0, 
-      root_mock_dir: /project/projectdirs/desi/mocks/mws/galaxia/alpha/v0.0.2/bricks,
+       # root_mock_dir: /project/projectdirs/desi/users/forero/galaxia_small,
```

Comment:
I commented out this testing location (forero/galaxia_small), but a small version of the galaxia catalog would be very useful.  Please move a copy with a version number to the standard /project/projectdirs/desi/mocks/ area, and update this commented entry to point to that.